src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/Chart/BitChartDemoColors.cs

```diff
@@ -6,6 +6,10 @@ namespace Bit.BlazorUI.Playground.Web.Pages.Components.Chart;
 
 public static class BitChartDemoColors
 {
+    public static Color GetRandomColor()
```

Comment:
it seems this method is not used anywhere.
if it's not used anywhere, please remove it.